site/content/en/docs/administration/advanced/update_guide.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Update guide'
+linkTitle: 'Update guide'
+weight: 11
+description: 'Instructions for updating CVAT deployed with docker compose'
```

Comment:
Let's call the process `upgrade`. Please rename 'update'/'updating' to 'upgrade'/'upgrading'.